### PR TITLE
Fix Mapillary Vistas data format

### PR DESCRIPTION
**Pull request by wonjuleee**
### Checklist

- [x] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [x] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/unit/test_mapillary_vistas_format.py
Comment (vinnamkim) [inline tests/unit/test_mapillary_vistas_format.py:1]: This file seems changing from unittest to pytest. Would you move this file to `tests/unit/data_formats` directory and make the test class to be inherited from `TestDataFormatBase`? You can refer to this file https://github.com/openvinotoolkit/datumaro/blob/develop/tests/unit/data_formats/coco/test_roboflow.py.

Comment (wonjuleee) [reply tests/unit/test_mapillary_vistas_format.py:1]: Thank you for letting me know this. I updated.

File: datumaro/plugins/data_formats/mapillary_vistas/base.py [hunk 1]
```diff
@@ -34,8 +33,21 @@
...
+        use_original_config=False,
+        keep_original_category_ids=False,
+        format_version="v2.0",
+        parse_polygon=False,
     ):
+        if format_version == "v1.2" and parse_polygon is True:
+            raise ImportError(
+                "Format version %s is not available for polygons. "
+                "Please try with v2.0 for parsing polygons." % format_version
+            )
```
Comment (JihwanEom) [inline datumaro/plugins/data_formats/mapillary_vistas/base.py]: How about to use f-string?
```suggestion
            raise ImportError(
                f"Format version {format_version} is not available for polygons. "
                "Please try with v2.0 for parsing polygons."
            )
```

File: datumaro/plugins/data_formats/mapillary_vistas/base.py [hunk 2]
```diff
@@ -180,80 +208,60 @@ def _load_instances_categories(self):
     def _load_instances_items(self):
         items = {}
 
-        instances_dir = osp.join(self._annotations_dir, MapillaryVistasPath.INSTANCES_DIR)
-        for instance_path in find_images(instances_dir, recursive=True):
-            item_id = osp.splitext(osp.relpath(instance_path, instances_dir))[0]
+        # class_dir = osp.join(self._annotations_dir, MapillaryVistasPath.CLASS_DIR)
```
Comment (JihwanEom) [inline datumaro/plugins/data_formats/mapillary_vistas/base.py:211]: Can we remove these comments?

File: datumaro/plugins/data_formats/mapillary_vistas/base.py [hunk 3]
```diff
@@ -48,7 +60,7 @@ def __init__(
             raise NotADirectoryError(
                 "Can't find annotation directory at %s. "
```
Comment (JihwanEom) [inline datumaro/plugins/data_formats/mapillary_vistas/base.py]: Same with above,
```suggestion
                f"Can't find annotation directory at {path}. "
```

File: datumaro/plugins/data_formats/mapillary_vistas/base.py [hunk 4]
```diff
@@ -48,7 +60,7 @@ def __init__(
...
                 "Expected one of these directories: %s"
```
Comment (JihwanEom) [inline datumaro/plugins/data_formats/mapillary_vistas/base.py]: Same with above,
```suggestion
                f"Expected one of these directories: {",".join(MapillaryVistasPath.ANNOTATIONS_DIRS)}"
```

File: datumaro/plugins/data_formats/mapillary_vistas/base.py (hunk 1 above)
Comment (wonjuleee) [reply datumaro/plugins/data_formats/mapillary_vistas/base.py]: I updated.

File: datumaro/plugins/data_formats/mapillary_vistas/base.py (hunk 3 above)
Comment (wonjuleee) [reply datumaro/plugins/data_formats/mapillary_vistas/base.py]: I updated.

File: datumaro/plugins/data_formats/mapillary_vistas/base.py (hunk 2 above)
Comment (wonjuleee) [reply datumaro/plugins/data_formats/mapillary_vistas/base.py:211]: I just want to leave this for the potential usage of class map.

File: datumaro/plugins/data_formats/mapillary_vistas/base.py (hunk 4 above)
Comment (wonjuleee) [reply datumaro/plugins/data_formats/mapillary_vistas/base.py]: I updated.

File: tests/unit/test_mapillary_vistas_format.py
Comment (wonjuleee) [reply tests/unit/test_mapillary_vistas_format.py:1]: I updated. Thanks.

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by JihwanEom — APPROVED** [review]

**Pull request merged**